eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/service/ProfileService.java

```diff
@@ -16,6 +17,7 @@
 
     @Autowired
     private ProfileRepository profileRepository;
+    private UserRepository userRepository;
```

Comment:
Erre is kell `@Autowired` ha nincs `@AllArgsConstructor`

Reply:
Én is támogatnám a Lombokos boilerplate generálást, jobban tesztelhető lesz a kód.